linkml-schema/kbmodel.yaml

```diff
@@ -299,27 +502,69 @@ classes:
     is_a: entity
     mixins:    
       - prov entity
+    in_subset:
+      - bican
+      - library_generation
+    slots:
+      - source barcode name
     attributes:
       cell prep type:
+        description: The type of cell preparation. For example Cells, Nuclei. This is a property of dissociated_cell_sample.
+        range: CellPrepType
+        required: true
       facs population plan: 
       num cells collected:
         range: integer
     slot_usage:
+      name: 
+        description: Name of a collection of dissociated cells or nuclei derived from dissociation of a tissue sample.
+        required: true 
+      source barcode name: 
+        description: >-
+          Name of oligo used in cell plexing. The oligo will tag allow separate dissociated cell samples to be combined downstream in the barcoded cell sample.
+          The oligo name is associated with a sequence in a lookup table. 
+          This sequence will be needed to during analysis, after alignment, to associate reads with parent dissociated cell sample.
+        required: true
       wasGeneratedBy:
         range: cell dissociation
       wasDerivedFrom:
         range: tissue sample
+        multivalued: true
+
 
 
   enriched cell sample:
     is_a: entity
     mixins:    
       - prov entity
+    in_subset:
+      - bican
+      - library_generation
+    slots:
+      - source barcode name
+    attributes:
+      histone modification marker:
+        description: >- 
+          Histone modification marker antibodies (eg H3K27ac, H3K27me3, H3K9me3) used in conjunction with an Enriched Cell Source Barcode in order to combine multiple Enriched Cell Populations before Barcoded Cell Sample step for 10xMultiome method. 
+          Each of the Histone antibodies captures an essential part of the epigenome.
+      population: 
+        description: >-
+          Actual percentage of cells as a result of using set of fluorescent marker label(s) to enrich dissociated_cell_sample with desired mix of cell populations. 
+          This plan can also be used to describe 'No FACS' where no enrichment was performed. This is a property of enriched_cell_prep_container.
+        required: true
     slot_usage:
-      wasGeneratedBy:
+      name:
+        description: >- 
+          Name of collection of enriched cells or nuclei after enrichment process (usually via FACS using the Enrichment Plan) applied to dissociated_cell_sample.
+        required: true
+      source barcode name:
+        description: >- 
+          Name of molecular barcode used to individual Enriched Cell Source to allow for pooling of Enriched Cell Sources before 10x load (Barcoding Cell step) [aka 1st round barcodes].
+          wasGeneratedBy:
```

Comment:
Think there's an indentation problem here

Reply:
@djarecka - isn't linting part of the action? if not we should.  i would think the linter would have caught this.

Reply:
@satra - linting is part of the action and it is passing with no errors so I think the indentation is fine.